change/@azure-communication-react-d74c9c67-8c1e-4d3c-a629-38ddfa99a3fb.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
this should be prerelease at this fix only applies to the call readiness features